lms/urls.py

```diff
@@ -206,8 +206,11 @@
         continue
 
     # Make the assumptions that the templates are all in the same dir
-    # and that they all match the name of the key (plus extension)
-    template = "%s.html" % key.lower()
+    # and that they all match the name of the key
+    template = key.lower()
+    if '.' not in template:
+        # Assume html if no file extension specified
+        template = "%s.html" % template
```

Comment:
Move ".html" into a setting -- alternatively keep the period here and just move "html"